### ISSUE TITLE
pretrained_word_embedding: missing + 1 for embedding matrix input

### ISSUE THREAD

**Issue by shafy** [keras-team/keras#8037]
Hi! Thanks for the nice examples. Just noticed that there is a small mistake in the `pretrained_word_embedding`example:

https://github.com/fchollet/keras/blob/130809e645e56d551250aef95c74f6f0f9463381/examples/pretrained_word_embeddings.py#L120

The input dim should be `num_words + 1`

**Comment by harshini-gadige** [comment]
@fchollet  -  PTAL
